src/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -50,68 +51,199 @@ internal enum SidNameUse
+        WinEnterpriseReadonlyControllersSid = 74,
+        /// <summary>Indicates a SID that matches an account read-only controllers group.</summary>
+        WinAccountReadonlyControllersSid = 75,
+        /// <summary>Indicates a SID that matches an event log readers group.</summary>
+        WinBuiltinEventLogReadersGroup = 76,
+        /// <summary>Indicates a SID that matches a read-only enterprise domain controller.</summary>
+        WinNewEnterpriseReadonlyControllersSid = 77,
+        /// <summary>Indicates a SID that matches the built-in DCOM certification services access group.</summary>
+        WinBuiltinCertSvcDComAccessGroup = 78,
+        /// <summary>Indicates a SID that matches the medium plus integrity label.</summary>
+        WinMediumPlusLabelSid = 79,
+        /// <summary>Indicates a SID that matches a local logon group.</summary>
+        WinLocalLogonSid = 80,
+        /// <summary>Indicates a SID that matches a console logon group.</summary>
+        WinConsoleLogonSid = 81,
+        /// <summary>Indicates a SID that matches a certificate for the given organization.</summary>
+        WinThisOrganizationCertificateSid = 82,
+        /// <summary>Indicates a SID that matches the application package authority.</summary>
+        WinApplicationPackageAuthoritySid = 83,
+        /// <summary>Indicates a SID that applies to all app containers.</summary>
+        WinBuiltinAnyPackageSid = 84,
+        /// <summary>Indicates a SID of Internet client capability for app containers.</summary>
+        WinCapabilityInternetClientSid = 85,
+        /// <summary>Indicates a SID of Internet client and server capability for app containers.</summary>
+        WinCapabilityInternetClientServerSid = 86,
+        /// <summary>Indicates a SID of private network client and server capability for app containers.</summary>
+        WinCapabilityPrivateNetworkClientServerSid = 87,
+        /// <summary>Indicates a SID for pictures library capability for app containers.</summary>
+        WinCapabilityPicturesLibrarySid = 88,
+        /// <summary>Indicates a SID for videos library capability for app containers.</summary>
+        WinCapabilityVideosLibrarySid = 89,
+        /// <summary>Indicates a SID for music library capability for app containers.</summary>
+        WinCapabilityMusicLibrarySid = 90,
+        /// <summary>Indicates a SID for documents library capability for app containers.</summary>
+        WinCapabilityDocumentsLibrarySid = 91,
+        /// <summary>Indicates a SID for shared user certificates capability for app containers.</summary>
+        WinCapabilitySharedUserCertificatesSid = 92,
+        /// <summary>Indicates a SID for Windows credentials capability for app containers.</summary>
+        WinCapabilityEnterpriseAuthenticationSid = 93,
+        /// <summary>Indicates a SID for removable storage capability for app containers.</summary>
+        WinCapabilityRemovableStorageSid = 94
```

Comment:
Can you please add comment to the end, saying that the code below has to be updated whenever we add a new value?